### PR TITLE
Gitbook configuration and audit log events document coversion to markdown

### PR DESCRIPTION
**Pull request by raits**
Created the structure files required by Gitbook. I also converted the audit log events document to markdown.

### CODE REVIEW

File: INTRODUCTION.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Depending on if you are joining an existing X-Road instance or creating a new one, the software packages you need to
+install vary quite a bit.
+
+In case you are looking to join an existing instance, be sure to familiarise yourself with the rules and documentation
+of the instance before moving on to the installation guides.
+
+If you are looking to set up your own instance, we recommend you visit the [X-Road website](https://x-road.global),
+have a look at the architecture documents on this page as well as familiarising yourself with the concepts with the
+free [X-Road Academy](https://x-road.thinkific.com) courses.
```
Comment (petkivim) [inline INTRODUCTION.md:36]: I would change this part a little bit:

> The X-Road software packages to be installed vary between different use cases.
>
> If you are joining an existing X-Road ecosystem, you should familiarise yourself with the ecosystem-specific documentation before moving to the Security Server installation guides. The X-Road ecosystem-specific documentation is provided and maintained by the ecosystem's X-Road Operator.
>
> Instead, if you're setting up a new X-Road ecosystem, it's strongly recommended to visit the [X-Road website](https://x-road.global/resources) for additional resources and study the architecture documents on this page. For a new X-Road ecosystem, it's required to set up the Central Server and a management Security Server.
>
> The free [X-Road Academy](https://academy.x-road.global/) courses provide online training for developers, users, operators, consultants, service providers and for anyone willing to learn more about X-Road.

File: doc/Architecture/spec-al_x-road_audit_log_events.md [hunk 2]
```diff
@@ -0,0 +1,465 @@
+# Audit log events
+
+Version: 1.9
+16.02.2023
+Doc. ID: SPEC-AL
```
Comment (petkivim) [inline doc/Architecture/spec-al_x-road_audit_log_events.md:5]: A line break seems to be missing from here in the `html` version.

File: doc/Architecture/spec-al_x-road_audit_log_events.md [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+| Delete backup file    | <ul><li>backupFileName - the name of the deleted backup file</li></ul>                       |
+| Restore configuration | <ul><li>backupFileName - the name of the backup file used to restore configuration</li></ul> |
+
+### 2.2 Security Server
+
+#### 2.2.1 Common Events
+
+The audit log events related to the UI logging.
+
+| Event        | Data fields |
```
Comment (petkivim) [inline doc/Architecture/spec-al_x-road_audit_log_events.md:328]: Is the "Data fields" column empty on purpose?

File: doc/Architecture/spec-al_x-road_audit_log_events.md [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+| API key create | <ul><li>apiKeyId - identifier of the API key</li><li>apiKeyRoles - array containing the roles associated with the API key</li></ul> |
+| API key update | <ul><li>apiKeyId - identifier of the API key</li><li>apiKeyRoles - array containing the roles associated with the API key</li></ul> |
+| API key remove | <ul><li>apiKeyId - identifier of the API key</li><li>apiKeyRoles - array containing the roles associated with the API key</li></ul> |
+
+#### 2.2.8 Technical Events
+
+The audit log events related to technical events, such as authentication failures. Except for Key management API log in
+event, these events are only logged if they fail.
+
+| Event                                                                                                         | Data fields |
```
Comment (petkivim) [inline doc/Architecture/spec-al_x-road_audit_log_events.md:440]: Is the "Data fields" column empty on purpose?

File: doc/Architecture/spec-al_x-road_audit_log_events.md [hunk 5]
```diff
@@ -0,0 +1,465 @@
...
+| Delete certificate <br><br> Event name is used if event fails before it is known where certificate is delete from (configuration or token) |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (petkivim) [inline doc/Architecture/spec-al_x-road_audit_log_events.md:403]: Is the "Data fields" column empty on purpose?

File: INTRODUCTION.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+To learn about setting up a Security Server cluster with an external load balancer, please take a look at the relevant
+documentation [here](doc/Manuals/LoadBalancing/ig-xlb_x-road_external_load_balancer_installation_guide.md).
+
+### Central Server
+
+The X-Road Central Server packages are currently only available for Ubuntu 20.04 and Ubuntu 22.04. You can find the
+installation manual [here](doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md).
+
+To learn about setting up a Central Server cluster, please also take a look at the relevant documentation
+[here](doc/Manuals/ig-csha_x-road_6_ha_installation_guide.md).
```
Comment (petkivim) [inline INTRODUCTION.md:59]: The test CA could be mentioned here at the end:

> Setting up a fully functional X-Road environment requires a Certificate Authority (CA) with an OCSP service and a time-stamping service. The [X-Road Test CA](ansible/TESTCA.md) can be used for testing and development purposes.

**Review by petkivim — CHANGES_REQUESTED** [review]

File: INTRODUCTION.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+# X-Road Documentation
+
+## What is X-Road
+
+X-Road® is open-source software and ecosystem solution that provides unified and secure data exchange between
+organisations.
+
+![X-Road overview](X-Road_overview.png)
```
Comment (petkivim) [inline INTRODUCTION.md:8]: The image should be updated to the latest version in `develop`:

https://github.com/nordic-institute/X-Road/blob/develop/X-Road_overview.png

File: doc/Architecture/spec-al_x-road_audit_log_events.md (hunk 2 above)
Comment (raits) [reply doc/Architecture/spec-al_x-road_audit_log_events.md]: Thanks, fixed. Also removed the date field, as we don't seem to use it in other documents anymore either.

File: doc/Architecture/spec-al_x-road_audit_log_events.md (hunk 3 above)
Comment (raits) [reply doc/Architecture/spec-al_x-road_audit_log_events.md:325]: Yes, it's the same way in the original docx.

File: doc/Architecture/spec-al_x-road_audit_log_events.md (hunk 5 above)
Comment (raits) [reply doc/Architecture/spec-al_x-road_audit_log_events.md:400]: Yes, it's the same way in the original docx for the `Delete Certificate` field.

File: INTRODUCTION.md (hunk 6 above)
Comment (raits) [reply INTRODUCTION.md:61]: Thanks, added.

File: INTRODUCTION.md (hunk 7 above)
Comment (raits) [reply INTRODUCTION.md:8]: Merged develop into the branch so that the latest changes are now available.

**Comment by petkivim** [comment]
Looks good to me.

**Review by petkivim — APPROVED** [review]

**Pull request merged**